styleguide/Components/ReactComponent/ReactComponent.js

```diff
@@ -28,7 +36,14 @@ const ReactComponent = ({ component, exampleMode }) => {
           ""
         )}`}
       >
-        <Caption>Github</Caption>
+        <Caption>Source</Caption>
+      </Link>
+      <Link
```

Comment:
Хочется сюда бахнуть какой-то разделитель, чтобы было очевиднее, что это две разные ссылки. Например:

```suggestion
      </Link> • <Link
```

А в остальном 👍🏻!